assets/app/scripts/controllers/overview.js

```diff
@@ -55,6 +55,10 @@ angular.module('openshiftConsole')
 
     watches.push(DataService.watch("services", $scope, function(services) {
       $scope.unfilteredServices = services.by("metadata.name");
+      var isEmpty = Object.keys($scope.unfilteredServices).length === 0;
+      $scope.renderOptions.showSidebarRight = !isEmpty;
+      $scope.showGetStarted = isEmpty;
```

Comment:
sorry to make you tweak it one more time... group this option under `renderOptions`?